x-pack/plugins/ml/public/file_datavisualizer/components/fields_stats/_field_stats_card.scss

```diff
@@ -0,0 +1,88 @@
+.card-container {
```

Comment:
Could any of these settings be used in the existing index based datavisualizer once that is converted to `sass`? Just wondering if some of these should be moved to a higher level `scss` file?

Reply:
yes, i imagine there will be common rules which can be moved to a shared location once the big sass conversion has happened